tools/CoqMakefile.in

```diff
@@ -796,7 +804,7 @@ define globvorule=
 # take care to $$ variables using $< etc
   $(1).vo $(1).glob &: $(1).v | $(VDFILE)
 	$(SHOW)COQC $(1).v
-	$(HIDE)$$(TIMER) $(COQC) $(COQDEBUG) $$(TIMING_ARG) $(COQFLAGS) $(COQLIBS) $(1).v
+	$(HIDE)$$(TIMER) $(COQC) $(COQDEBUG) $$(TIMING_ARG) $$(PROFILE_ARG) $(COQFLAGS) $(COQLIBS) $(1).v
```

Comment:
@SkySkimmer isn't there an extra dollar?

Reply:
No, this is like TIMER and TIMING_ARG, evaluation needs to be delayed as when globvorule is expanded the `$<` is not yet defined